src/main/java/nl/novi/Les11TechItEasy/Services/CiModuleService.java

```diff
@@ -0,0 +1,100 @@
+package nl.novi.Les11TechItEasy.Services;
+
+import nl.novi.Les11TechItEasy.Dto.input.CiModuleDto;
+import nl.novi.Les11TechItEasy.Dto.output.CiModuleOutputDto;
+import nl.novi.Les11TechItEasy.Exception.RecordNotFoundException;
+import nl.novi.Les11TechItEasy.Model.CiModule;
+import nl.novi.Les11TechItEasy.Repository.CiModuleRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class CiModuleService {
+    @Autowired
```

Comment:
Het is niet de bedoeling dat je @Autowired en constructor injection door elkaar gebruikt. p.s. @Autowired werkt ook niet op een final veld.